server/verify_test.go

```diff
@@ -1210,6 +1228,7 @@ func TestVerifyAttestationWithTdx(t *testing.T) {
 				Nonce:      nonce,
 				TrustedAKs: []crypto.PublicKey{ak.PublicKey()},
 				TEEOpts: &VerifyTdxOpts{
+					Validation:   TdxDefaultValidateOpts(nonce),
```

Comment:
We should have happy path without TEEOpts set as well

Reply:
Added